Maths/QuadraticFormula.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @function quadraticFormula
+ * @description This script will find the the roots of a quadratic equation.
+ * @param {number} a
+ * @param {number} b
+ * @param {number} c
+ * @return {string}
+ * @see https://en.wikipedia.org/wiki/Quadratic_formula
+ * @example quadraticFormula(1, -3, -4) = 'the roots are X1=4 and X2=-1'
+ * @example quadraticFormula(1, 5, 6) = 'the roots are X1=-2 and X2=-3'
+ * @example quadraticFormula(1, -3, 8) = 'the roots do not exist or the roots are imaginary'
+ */
+
+const quadraticFormula = (a, b, c) => {
```

Comment:
Please rename to `solveQuadraticEquation`

Reply:
Ok, no problem 👍